src/main/java/com/msclub/meetingmanager/service/MicrosoftTeamsService.java

```diff
@@ -74,7 +74,16 @@ public String createMicrosoftMeeting(String accessToken, MSTeamsInterviewDetails
 
         // create request body
         MicrosoftTeamsMeetingDetails microsoftTeamsMeetingDetails = new MicrosoftTeamsMeetingDetails();
-        microsoftTeamsMeetingDetails.setSubject("MS Club of SLIIT - Interview " + msTeamsInterviewDetails.getStudentName());
+
+        switch(type){
+            case INTERVIEW:
+                            microsoftTeamsMeetingDetails.setSubject("MS Club of SLIIT - Interview " + msTeamsInterviewDetails.getStudentName());
+                            break;
+            case INTERNAL_MEETING:
+                             microsoftTeamsMeetingDetails.setSubject("MS Club of SLIIT - Internal - Meeting " + msTeamsInterviewDetails.getStudentName());
```

Comment:
create a variable called title in microsoftTeamsMeetingDetails class  and replace getTitle something like that